server.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const server = express();
+server.use(express.json());
+
+let db = [
+    {
+        title: 'Final Fantasy VII',
+        genre: 'JRPG',
+        releaseYear: 1997
+    }
+];
+// server gets and server posts
+server.get('/games', (req, res) => {
+    res.status(200)
+        .json(db)
```

Comment:
The status will default to 200 if you send/json data over without a status method, so you could shorten this:
```javascript
  res.json(db)
```